client/src/Components/Card/EmissionsCard.js

```diff
@@ -1,35 +1,26 @@
 import React, { Component } from "react";
-import { graphql } from "react-apollo";
-import { getEmissionsQuery } from "../../queris/queris";
 import { Card, Icon } from 'semantic-ui-react';
-import LoadingSpinner from '../LoadingSpinner';
 
 
 class EmissionsCard extends Component {
-  displayCarData() {
-    let data = this.props.data;
-    if (data.loading) {
-      return <LoadingSpinner />;
-    } else {
-      return <div>{data.car.emissions}KG CO2</div>
-    }
-  }
+
   render() {
     return (
       <Card className="data-cards-r1" >
         <Card.Content>
-          <Icon float="left" circular inverted color="green" name="tree" size="big" />
+          <Icon circular inverted color="green" name="tree" size="big" />
           <div className="data-content">
-            <p>EMISSIONS</p>
-            <h2>{this.displayCarData()}</h2>
+            <p>Emissions</p>
+            <h2>{this.props.emission}KG C02</h2>
```

Comment:
2 should be subscript. https://www.w3schools.com/TAGS/tag_sub.asp 
http://www.climate-change-wisdom.com/CO2e.html

Reply:
KG should be `kg` http://www.simetricmatters.com/inkorrect.htm

Reply:
Fixed as follwoing
{this.props.emission}kg C0<sub>2</sub>